internal/logging/logging.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logging
+
+import (
+	"log"
+)
+
+const (
+	LevelNone  Level = iota
+	LevelDebug Level = iota
+	// Can add more levels in future if required...
+)
+
+type Level int
+
+// By default, nothing will be logged.
+var logLevel = LevelNone
```

Comment:
nit: consider using a `Logger` struct and a `func DefaultLogger()` this way we have the flexibility to reuse logger for more than `os.StdErr`

Reply:
you can get some inspiration from https://pkg.go.dev/log#Logger